qa327/__init__.py

```diff
@@ -11,6 +12,14 @@
 )
 
 app = Flask('this is a simple web application', template_folder=templates)
-app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///db.sqlite'
 app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
 app.config['SECRET_KEY'] = '69cae04b04756f65eabcd2c5a11c8c24'
+# if the user supplies a database file name, we use
+# that instead
+if len(sys.argv) > 1:
```

Comment:
This does not work in windows, the first argument is pytest, the second one is qa327_test.